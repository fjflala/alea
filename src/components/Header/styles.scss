.ui-header {
  width: 100%;
  padding: 16px;
  box-shadow: 1px 0 3px rgba($color: #333, $alpha: .4);
}