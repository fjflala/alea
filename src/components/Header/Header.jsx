import React from 'react';

import './styles.scss';

const Header = () => {
  return (
    <header className="ui-header">
      <figure>
        <img
          src="https://www.alea.com/wp-content/themes/alea/library/images/logo-alea.svg"
          alt="Alea"
        />
      </figure>
    </header>
  );
};

export default Header;
