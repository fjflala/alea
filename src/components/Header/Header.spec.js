import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

describe('<Header />', () => {
  it('should correctly render', () => {
    const tree = renderer
      .create(<Header />)
      .toJSON();
    
    expect(tree).toMatchSnapshot();
  });
});