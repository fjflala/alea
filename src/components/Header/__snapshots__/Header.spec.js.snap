// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should correctly render 1`] = `
<header
  className="ui-header"
>
  <figure>
    <img
      alt="Alea"
      src="https://www.alea.com/wp-content/themes/alea/library/images/logo-alea.svg"
    />
  </figure>
</header>
`;
